### DISCUSSION TITLE
Login data - saving

### DISCUSSION THREAD

**Discussion by Miquest** [NightscoutFoundation/xDrip#2340] [Q&A] [answered]
Dear xDrip community,
I'm a newbie in login systems and I want do develop a Dexcom Share follower. The software should get data every five minutes, so I need also login data every five minutes. How are the login Data for my Dexcom Account saved in xDrip+?
I would be happy if I could do it in the same way for my software!

Kind regards,
Thorin E.

**Comment by jamorham** [comment, marked as answer]
xDrip already has this functionality and the source code is published in the repository. This looks like a general programming question and probably something which shouldn't be attempted without sufficient experience.
